examples/nlp/language_modeling/conf/megatron_gpt_prompt_learning_config.yaml

```diff
@@ -59,10 +59,13 @@ model:
     total_virtual_tokens: 80 # Sum of tokens in virtual_token_splits must add to this number. Can differ between new and existing tasks, but must match across all new tasks being tuned at the same time.
     virtual_token_splits: [60, 20] # number of virtual tokens to be inserted at each VIRTUAL PROMPT location, must add to total_virtual_tokens
     truncate_field: 'passage' # The {field} in the prompt template whose text will be truncated if the input is too long, if null, inputs that are too long will just be skipped.
+    answer_only_loss: True 
+    answer_field: 'answer'
```

Comment:
shouldn't `answer_only_loss` always be `True`? I suggest remove it from the configure and make `loss_mask` only care about `answer` loss.

Reply:
Before the refactor, answer only loss = True didn't always give the best results when the prompt was short